### PR TITLE
Slack Channel Introduction / Description

### PR DESCRIPTION
**Pull request by sugaroverflow**
We thought it would be a good idea to have an introduction for people new to our slack channel. 
This is a first pass at that, feedback welcome! :)

(I added this to the working docs/ but let me know if there's a better place - thanks!)

### CODE REVIEW

**Comment by star-szr** [comment]
I really want to watch this now but here's the link for the New Orleans presentation :) https://events.drupal.org/neworleans2016/sessions/together-we-can-make-drupal-example-diversity-tech

**Comment by sugaroverflow** [comment]
Added, thank you @Cottser !

I've watched it a few times. It makes me feel less like a DD&I imposter.

File: working-docs/slack-channel-introduction.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# New Here? Walkthrough
+New to the Diversity & Inclusion slack channel?
+
+Here’s a couple of things you should know before you dive into the channel!
+
+## What is DD&I
+
+Drupal Diversity and Inclusion is an independent collective of volunteers who advocate for greater diversity in the community and more inclusive events, help create safe spaces, and build the capacity of allies.
+
+This group was created after DrupalCon New Orleans. Nikki Stevens & Karyn Cassio gave a presentation about diversity and inclusion. ++(link presentation page)++
```
Comment (cleverington) [inline working-docs/slack-channel-introduction.md]: > _++(link presentation page)++

https://events.drupal.org/neworleans2016/sessions/together-we-can-make-drupal-example-diversity-tech

**Comment by justAChris** [comment]
As a Noob to DD&I, I think this covers everything important. I'd consider promoting "We would like to:" as a header in kind of "What we do" so when people ask what the group does, it's easy to point to.

**Comment by sparklingrobots** [comment]
In the channel @zombree suggested a "Getting started with Github", which I think would be an excellent addition to this doc.  I suspect there is already such a document on the internet...

...okay, after a brief search, most of it seems to be "get started with your own repo".  But I found this video for how to use github issues: https://www.youtube.com/watch?v=TJlYiMp8FuY  But it's a start, until we have something more custom for our purposes?

File: working-docs/slack-channel-introduction.md (hunk 1 above)
Comment (sugaroverflow) [reply working-docs/slack-channel-introduction.md]: Thanks! I pushed commit 575bcbf, why didn't the PR update?

Comment (sugaroverflow) [reply working-docs/slack-channel-introduction.md]: Hmm maybe this is a conversation on the older commit version. The link is there now thank you!

**Comment by cleverington** [comment]
I wonder if it would be better to add this as a new page on the site?

Maybe under a link "Slackin' It" or "Slack" or something?

**Comment by sugaroverflow** [comment]
@cleverington that makes sense! 

I'm thinking if we put this in the working docs for now, I can add this info to the Getting Involved page on the website under a 'Slack Channel' heading?

The reason I want it in the working docs is to have a place to link to it from the slack channel's description :)

**Comment by sugaroverflow** [comment]
@justAChris Thank you for the feedback, it's super appreciated! I'm a little confused, where were you proposing it would be better to use "What we would like to do" as a heading? I like the idea, I'm just not sure where it would go. Instead of "Who We Are"?

**Comment by justAChris** [comment]
@sugaroverflow I misread it a bit. On initial read I thought that the “We would like to:” bullet points spoke more to that goals of the group instead of the purpose of the channel. Since this document is specifically talking about the slack channel, those ideas fit fine under “What this channel is for”, so the headings are great as is.

I would suggest we replace the “We would like to:” text since that comes off soft to me. Something like "In this channel we:” or “The purpose of this channel is to:"

File: working-docs/slack-channel-introduction.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+This group was created after DrupalCon New Orleans. Nikki Stevens & Karyn Cassio gave a [presentation about diversity and inclusion](https://events.drupal.org/neworleans2016/sessions/together-we-can-make-drupal-example-diversity-tech "presentation about diversity and inclusion").
+
+Nikki facilitated a follow up BOF was held the next day. Participants had so many ideas, and concerns and desires that we couldn’t do justice to most of them in the short amount of time we had. The general consensus seemed to be “Let’s keep talking about this so we can address everyone’s thoughts.” and so this group was created.
+
+## Who we are
+
+The DD&I team does not yet have a formal structure. We are a collective of volunteers who all believe in advocating for diversity and inclusion.
+
+We are not affiliated with the DA and our booth/stickers are all funded by donations.
```
Comment (justAChris) [inline working-docs/slack-channel-introduction.md]: > DA

Drupal Association

File: working-docs/slack-channel-introduction.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## What this channel is for
+
+Through this channel, we aim to to continue the conversation about diversity and inclusion within Drupal by providing a venue for important and sometimes difficult conversations that we need to have as a community.
+
+We would like to:
+- provide a safe (but moderated?) space to discuss issues of diversity and inclusion within the Drupal community.
```
Comment (justAChris) [inline working-docs/slack-channel-introduction.md]: > provide a safe (but moderated?) space

provide a moderated, safe space

**Review by justAChris — CHANGES_REQUESTED** [review]

File: working-docs/slack-channel-introduction.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+- provide support for people who are marginalized
+- advocate for a more open and diverse community whether through these conversations, presentations at camps and conferences, and BOF sessions.
```
Comment (johnshortess) [inline working-docs/slack-channel-introduction.md]: - Advocate for a more open and diverse Drupal community through Slack/IRC conversations, presentations and camps and conferences, BOF sessions, and other means

I started out by eliminating "whether" (which is superfluous), and got carried away. :)

Also, for consistency, all bullet items should start with a capital letter, and all should ether have or not have a period (or other appropriate punctuation) at the end.

File: working-docs/slack-channel-introduction.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- share resources/education about social justice issues in our technological communities - d&i, equality, inclusion, accessibility, etc.
+- provide strategies for increasing d&i of communities and organizations
+
+We would hope to inspiring others to help tell their stories and bring people together to make our community a place where more people can feel welcome.
```
Comment (johnshortess) [inline working-docs/slack-channel-introduction.md]: We hope to inspire others to help tell their stories and bring people together to make our community a place where more people can feel welcome.

**Comment by cleverington** [comment]
Added all recommendations.

**Comment by sugaroverflow** [comment]
Thanks @cleverington! I think this is ready for review :)

**Review by justAChris — APPROVED** [review]
Nothing further from me.

**Comment by sugaroverflow** [comment]
Note for self: need to add the participation and moderation guidelines as an update to this.

**Pull request merged**